File: app/src/main/java/kr/purplebeen/half/HALFService.java
package kr.purplebeen.half;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface HALFService {
    @GET("/users/add_user")
    Call<Result> registerDevice(@Query("id") String id, @Query("name") String name, @Query("rpi_ip") String rpiIp, @Query("hmd_ip") String hmdIp);
}

File: app/src/main/java/kr/purplebeen/half/HALFService.kt
package kr.purplebeen.half

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface HALFService {
    @GET("/users/add_user")
    fun registerDevice(@Query("id") id: String, @Query("name") name: String, @Query("rpi_ip") rpiIp: String, @Query("hmd_ip") hmdIp: String): Call<Result>

    @GET("/query")
    fun requestData(@Query("data") data : String) : Call<String>

}
